mantis-shaded/build.gradle

```diff
@@ -28,10 +28,10 @@ ext {
     jacksonVersion = '2.12.+'
     guavaFailureAccessVersion = '1.0.1'
     guavaVersion = '31.1-jre'
-    curatorVersion = '2.12.+'
-    zookeeperVersion = '3.4.+'
+    curatorVersion = '5.2.0'
+    zookeeperVersion = '3.5.9'
     jlineVersion = '0.9.94'
-    nettyVersion = '3.7.0.Final'
+    nettyVersion = '4.1.70.Final'
```

Comment:
umm, bump to `4.1.92.Final` since that looks to be the latest
artifactory - https://artifacts.netflix.net/nfrepo-everything-pom/io/netty/netty-all/